packages/presentation/lib/screen/login/login_screen.dart

```diff
@@ -132,13 +136,17 @@ class _LoginScreenState extends BlocScreenState<LoginScreen, LoginBloc> {
                       mainAxisAlignment: MainAxisAlignment.center,
                       children: [
                         LoginRegistrationButton(
-                          onPressed: bloc.logFacebook,
+                          onPressed: Responsive.isMobile(context)
+                              ? bloc.logFacebook
+                              : null,
                           buttonImage: ImagesPath.facebookImage,
                           buttonColor: AppColorsDark.facebookColor,
                         ),
                         const SizedBox(width: Dimens.size24),
                         LoginRegistrationButton(
-                          onPressed: bloc.logGoogle,
+                          onPressed: Responsive.isMobile(context)
+                              ? bloc.logGoogle
+                              : null,
                           buttonImage: ImagesPath.googleImage,
```

Comment:
maybe hide button when isn't mobile if(Responsive.isMobile(context)) LoginRegistrationButton()....

Reply:
Maybe I need to make another logic for this button for descktop, that's why I didn't hide button....